packages/node/src/policy-gen/to-policy.js

```diff
@@ -62,53 +69,72 @@ const inspectModuleRecords = (
 
     const { primordialMutations, strictModeViolations, dynamicRequires } =
       compatWarnings
-    if (primordialMutations.length) {
-      log.warning(
-        `Package "${moduleRecord.packageName}" contains potential SES incompatibilities (primordial mutations) at the following location(s):\n${primordialMutations
-          .map(({ node }) =>
-            chalk.yellow(
-              `- ${moduleRecord.file}:${node.loc.start.line}:${node.loc.start.column}`
-            )
-          )
-          .join('\n')}`
-      )
-    }
-    if (strictModeViolations.length) {
-      log.warning(
-        `Package "${moduleRecord.packageName}" contains potential SES incompatibilities (strict mode violations) at the following location(s):\n${strictModeViolations
-          .map(({ node }) =>
-            chalk.yellow(
-              `- ${moduleRecord.file}:${node.loc.start.line}:${node.loc.start.column}`
-            )
+    const nicePath = hrPath(moduleRecord.file)
+
+    /** @type {string[]} */
+    const warnings = perPackageWarnings.get(moduleRecord.packageName) || []
+
+    /**
+     * Adds SES compat issues to {@link warnings}
+     *
+     * @param {SesCompatObj[]} sesCompatObjs
+     * @param {string} type
+     */
+    const addWarnings = (sesCompatObjs, type) => {
+      if (sesCompatObjs.length) {
+        warnings.push(
+          ...sesCompatObjs.map(
+            ({
+              node: {
+                loc: {
+                  start: { line, column },
+                },
+              },
+            }) => {
+              const plainPath = stripVTControlCharacters(nicePath)
+              return `- ${chalk.yellowBright([plainPath, line, column].join(chalk.yellow(':')))} ${chalk.cyan(`(${type})`)}`
+            }
           )
-          .join('\n')}`
-      )
+        )
+      }
     }
-    if (dynamicRequires.length) {
+
+    addWarnings(primordialMutations, 'primordial mutation')
```

Comment:
- [ ] should be displayed to user so they know it will break. no "pleasant" solution